### PR TITLE
Add example for toi installation with helm chart dependencies

### PR DESCRIPTION
**Pull request by jensh007**
**What this PR does / why we need it**:
Add a new example under components to show how to deploy charts with dependencies

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release Notes**:

```feature user

```

### CODE REVIEW

File: components/subchartsdemo/README.md [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+          resource:
+            name: echo-chart
+          referencePath:
+          - name: echoserver
+```
+For echoserver a dependency is declared to a resource named `echo-chart` contained in the reference named `echoserver` of this component version (`referencePath`):
+
+```yaml
+components:
+- name: .../subcharts
```
Comment (mandelsoft) [inline components/subchartsdemo/README.md:31]: Wouldn't it be more readable if we put concrete names here?

File: components/subchartsdemo/README.md [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+The helm chart has multiple other charts as dependencies and has no templates. Its only purpose is to act as an anchor for deployment. The toi package is used to install the component with the helm-installer using the `ocm bootstrap` command.
+
+## Helm Charts with dependencies
+Goal of this deployment is to install two different components having their own helm-charts: echoserver and pod-info. Both have their own component version and a helm-chart resource. The chart dependencies are declared in the `packagespec.yaml` as `subcharts`:
+
+```yaml
+executors:
+  - ...
+    config:
+      ...
```
Comment (mandelsoft) [inline components/subchartsdemo/README.md:14]: I think a root chart is required here, also. ???

Comment (jensh007) [reply components/subchartsdemo/README.md]: full config added

File: components/subchartsdemo/README.md (hunk 1 above)
Comment (jensh007) [reply components/subchartsdemo/README.md]: component name ist now fully specified.

**Pull request merged**